mlsconverters/sklearn.py

```diff
@@ -3,61 +3,72 @@
 import json
 import numpy as np
 from scipy.stats._distn_infrastructure import rv_frozen
+from functools import partial
+
+
+# TODO why is this unused?
```

Comment:
coz we want to have a type specifier in the schema

Reply:
but it's not called or passed anywhere in the python... i don't understand

Reply:
mmmm i wonder what happened here as this should be called on every primitive value :O i'll look into it.